backend/lib/endpoints/schema/sendgrid.js

```diff
@@ -4,7 +4,10 @@ const { strictSchema } = require("./utils");
 const sendgridContactSchema = {
   body: strictSchema()
     .prop("email", S.string().format("email").required())
-    .prop("location", S.object().required()),
+    .prop("location", S.anyOf([S.object(), S.string()]))
```

Comment:
The reason I am doing this is because if the location is not set, then it is a string (per the way that it is on initial state) done by @CeesWang.

Reply:
We can set it to an empty object if the location is not provided.

Reply:
Yeah, working around this on the backend, cuz in the frontend, if I do that, it will break the filters you did.